src/composables/Cards.ts

```diff
@@ -453,6 +446,13 @@ export const Cards: Card[] = [
     base: 6,
     source: "King Doot (1/25)",
   },
+  {
```

Comment:
I see where you're coming from, but I still think it should mirror the position in game